### PR TITLE
S03_US04 Liste comptes Entreprise

### PR DESCRIPTION
**Pull request by JuMn88**
-creating relation OneToOne between User and Company
-updating UserFixtures
-new method in AdminController
-new template companiesList

### CODE REVIEW

**Review by endricklosange — DISMISSED** [review]

**Review by maxime-guichard — DISMISSED** [review]

File: src/Entity/User.php [hunk 1]
```diff
@@ -34,6 +34,26 @@ class User implements UserInterface
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $lastname;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private string $phone;
```
Comment (LucyMerlier) [inline src/Entity/User.php:50]: honest question : it seems like you put this field as required (not nullable in DB) => is it intentional?

File: templates/admin/companiesList.html.twig [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
+                        <li>
+                            <a class="dropdown-item" href="#"></a>
+                        </li>
+                    </ul>
+                </div>
+                <div class="col-lg-8 d-flex justify-content-center">
+                    <form class="form-inline mt-3 d-flex justify-content-start">
+                        <input class="form-control mr-sm-2 text-center" type="search" placeholder="Rechercher " aria-label="Search">
+                    </form>
```
Comment (LucyMerlier) [inline templates/admin/companiesList.html.twig:23]: it seems like this form is not used at all? => if it is not usable yet, just don't put it here

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/Entity/User.php (hunk 1 above)
Comment (JuMn88) [reply src/Entity/User.php:50]: Yes

**Review by maxime-guichard — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**